changelogs/fragments/44278-pamd_valid_simple_controls.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- lib/ansible/modules/system/pamd.py - Allow for validation of definitive control in pamd module.
```

Comment:
Use the module name without the extension rather than the full path, i.e., `- pamd -`.